students/MikeShand/lesson03/listlab1.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+#Series 1
+#Here's the initial list
+
+fruit=['Apples','Pears','Oranges','Peaches']
+print(fruit)
+
+#This prompts the user to add a fruit, then makes a new list and prints the result
+
+n=input('what fruit would you like to add?')
+# fruit2=[n]
+#fruit3=fruit+fruit2
+#print(fruit3)
+
+fruit.append(n)
+print(fruit)
+
+#Now the user selects a number, and this function displays the associated fruit
+
+x=int(input('what number fruit would you like to see?'))
+#if x<=len(fruit) and x!=0:
+if 0<x<=len(fruit):
+    print(x, fruit[x-1])
+else:
+    print("Invalid selection")
+
+#This demonstrates adding a user defined fruit with +
+
+y=input('what other fruit would you like to add?')
+#fruit4=[y]
+#fruit5=fruit4+fruit3
+#print(fruit5)
+
+fruit.append(y)
+print(fruit)
+
+#Now we'll add a user defined fruit with the insert operation
+
+z=input('Come on! Add one more fruit!')
+fruit.insert(0,z)
+
+#Here we use startswith to print a list of fruits that start with P
+
+for i in fruit:
+    if i.startswith('P'):
+        print('Here are your P fruits:',i)
+
+#Series 2
+
+fruit=['Apples','Pears','Oranges','Peaches']
+
+print('Now lets start series two')
+print("Here's the same series again")
+print(fruit)
+print("And here it is with the last fruit removed")
+fruit.remove(3)

```

Comment:
`remove` method will remove by value. looks like you tried to remove by index?

list has a `pop` method that will remove an item at any index (by default removes last index)

```
>>> fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
>>> fruit.pop()
'Peaches'
>>> fruit
['Apples', 'Pears', 'Oranges']
>>> fruit.pop(0)
'Apples'
>>> fruit
['Pears', 'Oranges']

```